docs/backends/Google.md

```diff
@@ -202,6 +202,33 @@ If the Docker image to be pulled is not public the `docker pull` will fail which
 If using any of these private Docker workflow options it is advisable to add
 them to the `workflow-options.encrypted-fields` list in Cromwell configuration.
 
+**Runtime Compute Engine configuration**  
+_runtime_ section of the WDL task contains information about the required computational resources.
+```
+runtime {
+    cpu: CPU_COUNT
+    memory: MEM_SIZE
+    cpuPlatform: PLATFORM
+  }
+```
+By default Cromwell creates custom type instances:  
```

Comment:
Probably this is not something that should be in the cromwell docs. If you think that this should be pointed out then please move it to the pull request description.

Reply:
In the docs you could say that cromwell supports both `n1` and `n2` instances in both custom and predefined configurations.